src/engine/effect_functions.asm

```diff
@@ -763,4 +763,35 @@ Func_2cbfb: ; 2cbfb (b:4bfb)
 	ret
 ; 0x2cc0a
 
-	INCROM $2cc0a, $30000
+	INCROM $2cc0a, $2f4e1
+	
+ImpostorProfessorOakEffect: ; 2f4e1 (b:74e1)
```

Comment:
While `Impostor` is a valid spelling, it is spelled `Imposter` in this game

Reply:
Changed the spelling in the most recent commit